common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -2896,6 +2911,75 @@ object CelebornConf extends Logging {
       .longConf
       .createOptional
 
+  val WORKER_JVM_QUAKE_ENABLED: ConfigEntry[Boolean] =
+    buildConf("celeborn.worker.jvm.quake.enabled")
+      .categories("worker")
+      .version("0.3.2")
+      .doc("When true, Celeborn worker will start the jvm quake to monitor of gc behavior, " +
+        "which enables early detection of memory management issues and facilitates fast failure.")
+      .booleanConf
+      .createWithDefault(false)
+
+  val WORKER_JVM_QUAKE_CHECK_INTERVAL: ConfigEntry[Long] =
+    buildConf("celeborn.worker.jvm.quake.check.interval")
+      .categories("worker")
+      .version("0.3.2")
+      .doc("Interval of gc behavior checking for worker jvm quake.")
+      .timeConf(TimeUnit.MILLISECONDS)
+      .createWithDefaultString("1s")
+
+  val WORKER_JVM_QUAKE_RUNTIME_WEIGHT: ConfigEntry[Double] =
+    buildConf("celeborn.worker.jvm.quake.runtime.weight")
+      .categories("worker")
+      .version("0.3.2")
+      .doc(
+        "The factor by which to multiply running JVM time, when weighing it against GCing time. " +
+          "\"Deficit\" is accumulated as gc_time - runtime * runtime_weight, and is compared against threshold " +
+          "to determine whether to take action.")
+      .doubleConf
+      .createWithDefault(5)
+
+  val WORKER_JVM_QUAKE_DUMP_THRESHOLD: ConfigEntry[Long] =
+    buildConf("celeborn.worker.jvm.quake.dump.threshold")
+      .categories("worker")
+      .version("0.3.2")
+      .doc("The threshold of heap dump for the maximum GC \"deficit\" which can be accumulated before jvmquake takes action.")
+      .timeConf(TimeUnit.MILLISECONDS)
+      .createWithDefaultString("30s")
+
```

Comment:
Should it be stated that if kill.threshold is smaller than dump.threshold, no dump will be generated？